autogen/agentchat/groupchat.py

```diff
@@ -384,6 +384,12 @@ def run_chat(
                     raise
             if reply is None:
                 break
+
+            # check for "clear history" phrase in reply and activate clear history function if found
```

Comment:
I suggest adding:
1. A field in GroupChat about whether to enable the clear_history command, with a default value False, and a docstr to suggest that this is an experimental feature. The reason is that a user may not be aware of this behavior and it can become a hidden bug when "CLEAR HISTORY" is used for a different purpose. An example should be provided in the docstr to explain how to use it.
2. A check here to enable the clearing history only if the field is set to True.

Reply:
ok, working on it

Reply:
Done, please see

Reply:
Thanks. Why is the "role" check removed? Also, the two "if" can be merged into one.